src/main.ts

```diff
@@ -1,16 +1,19 @@
-import * as core from '@actions/core';
-import { ToolsDirectory } from './toolsDirectory';
-import * as dotnet from './dotnet';
-import * as cake from './cake';
-import { CakeArgument } from './cakeParameter';
+import * as core from "@actions/core";
+import { ToolsDirectory } from "./toolsDirectory";
+import * as dotnet from "./dotnet";
+import * as cake from "./cake";
+import { CakeArgument } from "./cakeParameter";
+import yargsParser from "yargs-parser";
```

Comment:
The convention in the codebase is to use single quotes.

```suggestion
import yargsParser from 'yargs-parser';
```